### DISCUSSION TITLE
Solicitação de um email oficial para a Python Norte

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#124] [Geral]
Atualmente a Python Norte norte utiliza o email <pynorteam@gmail.com>. Gostaria de solicitar o acesso ao email <norte@python.org.br>, para que possamos passar mais credibilidade para o evento.

**Comment by lucianoratamero** [comment]
talvez a @naanadr possa ajudar?

**Reply by naanadr** [reply]
É comigo mesmo 😃

**Comment by naanadr** [comment]
@MateusSantosMeuBem até o domingo devo gerar o email para vocês.

**Reply by naanadr** [reply]
@MateusSantosMeuBem foi enviado um e-mail para o `pynorteam@gmail.com` com um link para vocês redefinem a senha de acesso do `norte@`, o link tem validade de 48h. Por favor, configurem verificação de 2 fatores 😄 

Quando estiver tudo certo, me avisa.

**Reply by meu-bem** [reply]
Oi, @naanadr! Valeu pela ajuda. Já configurei as coisas por aqui.

**Comment by rougeth** [comment]
@naandr acho uma boa configurar um tempo de expiração para os e-mails usados pelos eventos (talvez um ano?). Tem essa opção no Google Admin: https://support.google.com/a/answer/139399?hl=pt-BR

- Na seção Validade, selecione o período de validade das senhas.

Observação: se uma conta de usuário adicionou um usuário delegado, ele ainda pode acessar a conta com uma senha expirada. Para impedir o acesso contínuo, redefina a senha da conta ou [remova esse usuário](https://support.google.com/mail/answer/138350#remove_delegate).

**Reply by naanadr** [reply]
@rougeth adicionei tempo de expiração das senhas de 1ano e obriguei a serem cadastradas senhas fortes. Agora sobre os usuários delegados, não ficou muito claro se nessas redefinições de senha obrigatórias que o pessoal vai ter que fazer, os acessos dos usuários delegados serão removidos. Não conhecia esse negócio 🤔

**Reply by rougeth** [reply]
Eu também não conhecia. Talvez seja o caso de desabilitar a permissão dos usuários de adicionarem usuários delegados? Não sei se esse é um caso de uso para os eventos.

**Reply by naanadr** [reply]
Configuração desabilitada (segui essa [docs](https://support.google.com/a/answer/7223765?hl=pt-BR)) para todos os e-mails dentro do nosso Workspace:
![image](https://github.com/user-attachments/assets/27eb2544-a476-4d9e-aa12-548c849bcee0)

Obrigada pelas dicas @rougeth ❤️
